src/scripts/main.js

```diff
@@ -1,3 +1,38 @@
 'use strict';
 
-// write your code here
+const logo = document.querySelector('.logo');
+
+function createPromise(message) {
+  return new Promise((resolve, reject) => {
+    if (!message) {
+      resolve('Promise was resolved!');
+    } else {
+      setTimeout(() => reject(message), 3000);
+    }
+  });
+}
```

Comment:
This function `createPromise` creates a promise that either resolves immediately or rejects after 3 seconds. However, it's not clear why you are checking for the `message` parameter in line 7. If you want to create a function that returns a promise which either resolves or rejects based on some condition, you should make that condition explicit in the function parameters, rather than implicitly depending on whether a string is passed in or not.